root/etc/e-smith/templates/etc/nethcti/services.json/10base

```diff
@@ -1,6 +1,11 @@
 {
   my $ports_str = ${'nethcti-server'}{TCPPorts};
   our @ports = split(',', $ports_str);
+  if (${'nethcti-server'}{tlsSecureOptions} eq "true") {
+    our $tlsSecureOptions = "true";
+  } else {
+    our $tlsSecureOptions = "false";
+  }
```

Comment:
Since you're keeping the same value of the prop, you can simplify it.
Also make sure there is a safe default:
```suggestion
  our $tlsSecureOptions = ${'nethcti-server'}{tlsSecureOptions} || "false";
```